modules/openapi-generator/src/test/java/org/openapitools/codegen/python/PythonClientExperimentalTest.java

```diff
@@ -289,9 +289,8 @@ public void mapModelTest() {
         Assert.assertEquals(cm.classname, "sample.Sample");
         Assert.assertEquals(cm.description, "a map model");
         Assert.assertEquals(cm.vars.size(), 0);
-        Assert.assertEquals(cm.parent, "dict");
-        Assert.assertEquals(cm.imports.size(), 1);
-        Assert.assertEquals(Sets.intersection(cm.imports, Sets.newHashSet("children.Children")).size(), 1);
```

Comment:
Why did we remove the imports check here?

Reply:
> Why did we remove the imports check here?

Because now that composed schemas support the `additionalProperties` keyword, the additional properties are no longer in the parent. The other unit tests cover the scenario of composed schema with additional properties.

Reply:
Okay, but  what should we expect to see in cm.imports here?
An empty list?
If it a list fo length 1, what should it be?
Can you put the assertions back in verifying that the length and value of cm.imports is correct?

Reply:
It is there: https://github.com/OpenAPITools/openapi-generator/pull/6290/files/89117b9b54879154459c08fe790586cf6e03fbb8#diff-81a58419c148eb8bb7ddfa31ed3f0d2cR292